gb.go

```diff
@@ -55,23 +54,29 @@ func copyfile(dst, src string) error {
 	return err
 }
 
-func run(dir, command string, args ...string) error {
-	var buf bytes.Buffer
-	err := runOut(&buf, dir, command, args...)
-	if err != nil {
-		fmt.Printf("# %s %s\n%s", command, strings.Join(args, " "), buf.String())
-	}
-	return err
+// RunErr is returned when an exec.Command fails
+type RunErr struct {
+	Err    error
+	Args   []string
+	Output string
 }
 
-func runOut(output io.Writer, dir, command string, args ...string) error {
+func (e RunErr) Error() string {
+	s := fmt.Sprintf("Run Error: %v\n", e.Err)
+	s += fmt.Sprintln("# ", strings.Join(e.Args, " "))
+	s += e.Output
+	return s
+}
+
+func run(dir, command string, args ...string) error {
 	cmd := exec.Command(command, args...)
 	cmd.Dir = dir
-	cmd.Stdout = output
-	cmd.Stderr = os.Stderr
-	// Debugf("cd %s; %s", cmd.Dir, cmd.Args)
-	fmt.Println(strings.Join(cmd.Args, " "))
```

Comment:
This `fmt.Println` sneaked in from the cgo changes, I think. I changed it back to `Debugf` which seems more in line with the _be less noisy_ discussion.